src/AutoDelete.sh

```
@@ -0,0 +1,56 @@
+    # exit code will prematurely exit the script; if no error was
+    # reported in the job_error.json file, then the failure reason
+    # will be AppInternalError with a generic error message.
+    # The following line(s) use the dx command-line tool to upload your file
+    # outputs after you have created them on the local file system.  It assumes
+    # that you have used the output field name for the filename for each output,
+    # but you can change that behavior to suit your needs.  Run "dx upload -h"
+    # to see more options to set metadata.
+
+    sudo -H python3 -m pip install --no-index --no-deps packages/*
+    
+    dx download "$config_file" -o config_file
```

Shellcheck (0.10.0)

[warning] config_file is referenced but not assigned.

(SC2154)